pkgs/servers/web-apps/moodle/moodle-utils.nix

```diff
@@ -8,15 +8,16 @@ let
     configurePhase ? ":",
     buildPhase ? ":",
     buildInputs ? [ ],
+    nativeBuildInputs ? [ ],
     ...
   }:
   stdenv.mkDerivation (a // {
     name = name;
 
     inherit pluginType;
-    inherit configurePhase buildPhase;
+    inherit configurePhase buildPhase buildInputs;
```

Comment:
`buildInputs = []` is already the default

Reply:
I think this should be reverted.